### DISCUSSION TITLE
GitHub Student Developer Pack Issue

### DISCUSSION THREAD

**Discussion by abdulrahmanalshabo** [community/community#57916] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Hi I have been approved 1 day ago but the benefits Didn't open
![ff42](https://github.com/community/community/assets/136307479/bf262ea6-ccdf-4077-b5fd-17a634535f25)

**Comment by jge162** [comment, marked as answer]
It takes 24-72 hours to show up.
